tests/unit/introspection.tcl

```diff
@@ -233,6 +233,27 @@ start_server {tags {"introspection"}} {
         r client list
     } {*name= *}
 
+    test {Coverage: Basic CLIENT CACHING} {
+        set rd_redirection [redis_deferring_client]
+        $rd_redirection client id
+        set redir_id [$rd_redirection read]
+        r CLIENT TRACKING on OPTIN REDIRECT $redir_id
+        r CLIENT CACHING yes
+        r CLIENT TRACKING off
+    } {OK}
+
+    test {Coverage: Basic CLIENT REPLY} {
+        r CLIENT REPLY on
+    } {OK}
+
+    test {Coverage: Basic CLIENT TRACKINGINFO} {
+        r CLIENT TRACKINGINFO
+    } {flags off redirect -1 prefixes {}}
+
+    test {Coverage: Basic CLIENT GETREDIR} {
+        r CLIENT GETREDIR
+    } {-1}
+
```

Comment:
aren't these covered by tracking.tcl?
or at least maybe they should be moved there?

Reply:
yes, they are, but we skip tracking.tcl when running with --log-req-res (can't remember why but there must be a good reason)

Reply:
so maybe move them to tracking.tcl, but without the "tracking" tag, and add a comment as to why...?